### ISSUE TITLE
Useful API for TabSheet

### ISSUE THREAD

**Issue by cdir** [vaadin/vaadin-tabs#178]
I like the abstraction of Tabs in contrast to always have a TabSheet that contains the Tab-Bar and the page. But in most cases where I use Tabs I want to have a TabSheet, that means I want to switch a page when I select a tab. There should be a useful TabSheet API as well to simply create a tab next to the corresponding page. Of course I could write this piece of code in my own application only once but why isn't there a useful default implementation?

**Comment by web-padawan** [comment]
Closing as duplicate of #94 as both issues are now in the same repository.
